metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala

```diff
@@ -406,6 +425,70 @@ final class FileDecoderProvider(
     }
   }
 
+  private def decodeCFRFromClassFile(
+      path: AbsolutePath
+  ): Future[DecoderResponse] = {
+    Future {
+      Try {
+        val out = new ByteArrayOutputStream()
+        val err = new ByteArrayOutputStream()
+        val exceptions = new ListBuffer[Exception]()
+        val psOut = new PrintStream(out)
+        val psErr = new PrintStream(err)
+        try {
+          val sink = new OutputSinkFactory() {
+
+            override def getSupportedSinks(
+                sinkType: OutputSinkFactory.SinkType,
+                sinkClasses: Collection[OutputSinkFactory.SinkClass]
+            ): ju.List[OutputSinkFactory.SinkClass] =
+              List(
+                OutputSinkFactory.SinkClass.EXCEPTION_MESSAGE,
+                OutputSinkFactory.SinkClass.STRING
+              ).asJava
+
+            override def getSink[T](
+                sinkType: OutputSinkFactory.SinkType,
+                sinkClass: OutputSinkFactory.SinkClass
+            ): OutputSinkFactory.Sink[T] =
+              sinkType match {
+                case OutputSinkFactory.SinkType.JAVA => psOut.print(_)
+                case OutputSinkFactory.SinkType.EXCEPTION =>
+                  _ match {
+                    case msg: SinkReturns.ExceptionMessage =>
+                      exceptions += msg.getThrownException()
+                    case f => psErr.print(f)
```

Comment:
I'm wondering, in which case this branch is used?
Could it covered by a test?

Reply:
I can't trigger it so I've removed it